src/core/tests/frontend/paddle/places.cpp

```diff
@@ -2,8 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <frontend/shared/include/utils.hpp>
 #include <openvino/frontend/manager.hpp>
+#include <utils.hpp>
```

Comment:
utils is a very common name. I think will be better to have <frontend/shared/include/utils.hpp>.

Can we add additional subfolders to an include folder of shared tests in order to achive such result?